homeassistant/components/lcn/const.py

```diff
@@ -180,7 +178,7 @@
     "PERCENT",
     "PPM",
     "VOLT",
-    ELECTRIC_POTENTIAL_VOLT,
+    UnitOfElectricPotential.VOLT,
```

Comment:
This list hints at a bad design pattern.

The entity should always return the Home Assistant units as far as possible.

If we get a unit from the lcn device there should be a map between the lcn units and the Home Assistant units. If there's no device unit available and only a user configuration, we should only allow the user to pick Home Assistant units.

Reply:
I agree with the bad design pattern. That is why I originally didn't add more units.
Having thought some more about it, I think at this stage it is better to remove all Home Assistant units, and revisit it with the involvment of the code owner.

Reply:
Ok.